### ISSUE TITLE
How to loop through the array in the variable like a list

### ISSUE THREAD

**Issue by ilyesedina** [ilyesedina/Terraform#21]

**Comment by ilyesedina** [comment]
Lab 9
code line 177

**Comment by ilyesedina** [comment]
Error: 
│ Error: creating EC2 Subnet: InvalidSubnet.Conflict: The CIDR '10.0.250.0/24' conflicts with another subnet
│       status code: 400, request id: cf01f3c4-0213-4c44-947c-8a65f0ead8dd
│
│   with aws_subnet.list_subnet["dev"],
│   on main.tf line 177, in resource "aws_subnet" "list_subnet":
│  177: resource "aws_subnet" "list_subnet" {

Solution: 
Re run the `terraform apply` since the cider block didn't had enough time to recrate the resource

**Comment by ilyesedina** [comment]
[Closing Issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) form commit message:
```
<html>
<body>
<!--StartFragment-->
Linked issue | Syntax | Example
-- | -- | --
Issue in the same repository | KEYWORD #ISSUE-NUMBER | Closes #10
Issue in a different repository | KEYWORD OWNER/REPOSITORY#ISSUE-NUMBER | Fixes octo-org/octo-repo#100
Multiple issues | Use full syntax for each issue | Resolves #10, resolves #123, resolves octo-org/octo-repo#100

<!--EndFragment-->
</body>
</html>
```
